packages/terra-date-picker/src/DateInput.jsx

```diff
@@ -104,9 +103,29 @@ class DatePickerInput extends React.Component {
   constructor(props) {
     super(props);
 
+    const format = DateUtil.getDateFormat(props.intl.formatMessage({ id: 'Terra.datePicker.dateFormatOrder' }));
```

Comment:
Linter rule on Line 101 can now be removed.

Reply:
Removed! [89c3693](https://github.com/cerner/terra-framework/pull/1004/commits/89c3693f31162adfc53f90f681cdfda21588b6b0)